### PR TITLE
0.11.0 plat 5416 restart live proxy

### PR DESCRIPTION
**Pull request by gadyaari**

### CODE REVIEW

File: lib/entry/LiveEntry.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ function LiveEntry(entryObject, isNewSession) {
     events.EventEmitter.call(this);
     this.entryObject = entryObject;
     this.entryId = entryObject.entryId;
+    this.recovered = entryObject.liveStatus === 1;
```
Comment (kalturaguy) [inline lib/entry/LiveEntry.js]: don't put here 1 but the ENUM value

File: lib/BackendClient.js [hunk 2]
```diff
@@ -154,32 +153,49 @@ var BackendClient = (function(){
...
-                else
-                    liveEntry.serverType = kalturaTypes.KalturaEntryServerNodeType.LIVE_BACKUP;
+            if (results[0].objectType !== "KalturaAPIException" && results[1].objectType !== "KalturaAPIException") {  //Use to indicate success operation
+                var liveEntry = parseServerEntryInfo(serverNodeId, results);
                 deferred.resolve(liveEntry);
             }
             else {
                 logger.error("Failed to retrieve live stream entry info for from server: %j", result);
-                deferred.reject(new Error("%s: %s", result.objectType, result.code));
+                deferred.reject(new Error("%s: %s", result[0].objectType, result[0].code));
```
Comment (kalturaguy) [inline lib/BackendClient.js]: please test

File: lib/BackendClient.js [hunk 3]
```diff
@@ -154,32 +153,49 @@ var BackendClient = (function(){
...
+        client.entryServerNode.listAction(null, entryServerNodeFilter);
+
         var startTime = new Date();
-        client.liveStream.get(function(result, err, headers) {
+        client.doMultiRequest(function(results, err, headers) {
             var endTime = new Date();
             var operationTime = endTime - startTime;
             logger.info("Got server node info for entry %s (took %d)", entryId, operationTime);
-            printAPIResponse(logger, result, headers);
+            printAPIResponse(logger, results[0], headers);
```
Comment (kalturaguy) [inline lib/BackendClient.js]: print also results [1]  (in printAPiResponse)

File: lib/BackendClient.js [hunk 4]
```diff
@@ -154,32 +153,49 @@ var BackendClient = (function(){
...
 
-    function getEntryInfo(entryId, serverNodeId) {
+    function getServerType(serverNodeId, resultReceived) {
+        return (resultReceived.primaryServerNodeId === serverNodeId) ? kalturaTypes.KalturaEntryServerNodeType.LIVE_PRIMARY :
+            kalturaTypes.KalturaEntryServerNodeType.LIVE_BACKUP;
+    }
+
+    function parseServerEntryInfo(serverNodeId, apiCallResults) {
+        var liveEntryObject = convertKalturaEntry(apiCallResults[0]);
+        liveEntryObject.serverType = getServerType(serverNodeId, apiCallResults[0]);
```
Comment (kalturaguy) [inline lib/BackendClient.js]: @yossipapi  shouldn't we take the serverType from serverEntryNode instead of liveEntry?

Comment (gadyaari) [inline lib/BackendClient.js]: done. He said NO!

File: lib/BackendClient.js (hunk 3 above)
Comment (gadyaari) [inline lib/BackendClient.js]: done

**Pull request closed without merging**